cves/CVE-2005-3352.yml

```diff
@@ -55,10 +59,8 @@ fixes:
    - commit: 9de8a693d16e5ce76ff5b7f727311e783a51f34a
      note:
 vccs:
-  - commit: 9de8a693d16e5ce76ff5b7f727311e783a51f34a
-    note: Rename from mod_imap
-  - commit: a83b21763cd2fbfd97bf5cbb5f01ea4d7e7ac3c3
-    note:
+   - commit: Patch for apache-1.3 tree (ack fielding, jorton)/apache-2.0 tree (ack fielding, jorton) - Can't find commit ID     
```

Comment:
this might be failing the build, should throw this in the note and leave commit blank

Reply:
I agree with that, you can only put a commit ID there.

Reply:
I just removed the indentations and the build passed. I appreciate the help.